### ISSUE TITLE
"Lyrics provider" is missing under Settings > General

### ISSUE THREAD

**Issue by Gamartto** [ytmdesktop/ytmdesktop#244]
Hi, 

I just installed this software and upon doing the first time configuration, basically checking all the settings, I just found out that the "lyrics provider" option (the one you see in the readme.md screenshot) is not present.

Is this normal?
App Version: 1.11.0
OS: Win10 1903

If this is normal behavior, Is there a way to change the lyrics provider?

Thanks.

**Comment by adlerluiz** [comment]
Hello @Gamartto before it was necessary to define one of the available providers, now it searches in 3 different providers automatically in cascade, if it does not find in the first, it searches in the second, if it does not find in the second, it searches in the third

**Comment by rickinbsb** [comment]
this option was removed on 1.11 because the app now cycles on all of them, checking again in another source if the song wasn't found

**Comment by Gamartto** [comment]
Thanks for the replies. 

I have 3 follow up questions before closing this post.

Is it possible to know which lyric providers are being used? 
Can I add more manually via a config file?
Can I change (increase/decrease) the font size?

Thanks again.

**Comment by adlerluiz** [comment]
1. on the bottom of lyrics window 
![image](https://user-images.githubusercontent.com/2112638/87331608-a1079a00-c510-11ea-9c67-1ac3ccbcfa90.png)

2. no, because each letter system has a way to return the letter, it is not standardized and also the search form, do you have any that you want to be added?

3. No, but it's a good one and I can add

**Comment by Gamartto** [comment]
Usually when I search for lyrics google returns the lyrics from LyricFind. And that sources seems to have pretty much all the lyrics and without mistakes.

**Comment by rickinbsb** [comment]
this seems to be a paid service

**Comment by ZombieAnidata** [comment]
I just found a problem with this it should have a button to switch the lyrics because the lyrics sometimes are wrong and you can't get from another source or they are romanized or the live lyrics bug because they are wrong and when i scroll down it will scroll up.

**Comment by adlerluiz** [comment]
unfortunately I have seen this happening too and I am considering other providers, currently, he checks in the first, if I haven't found it, searches in the second, and if I also can't find it, search in the third

**Comment by SaveliyKolesnikov** [comment]
@adlerluiz Maybe we can upload lyrics from 3 supported now sources and add a dropdown that will contain sources that have lyrics?

**Comment by adlerluiz** [comment]
previously in configurations it could be selected between 2 providers, however I added one more and left it automatic, it checks in the first, if it doesn't find, searches in the second and if it doesn't find, searches in the third ...

I recently saw that YouTube Music will implement the lyrics in the web version and I will be able to get it from there too,

if the option is added, I can think of something like that
[providers list on settings]
1. YTMusic
2. Provider X
3. Provider Y
...
